constants/cardColorArray.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [
+  { COLOR_ID: 1, COLOR_PRIMARY: "#DB1212", COLOR_SECONDARY: "#F88181" },
+  { COLOR_ID: 2, COLOR_PRIMARY: "#F4A261", COLOR_SECONDARY: "#FFBF8C" },
+  { COLOR_ID: 3, COLOR_PRIMARY: "#2A9D8F", COLOR_SECONDARY: "#42E0CD" },
+  { COLOR_ID: 4, COLOR_PRIMARY: "#165692", COLOR_SECONDARY: "#3D8BD3" },
+  { COLOR_ID: 5, COLOR_PRIMARY: "#E9C46A", COLOR_SECONDARY: "#FFE094" },
+  { COLOR_ID: 6, COLOR_PRIMARY: "#264653", COLOR_SECONDARY: "#387892" },
+  { COLOR_ID: 7, COLOR_PRIMARY: "#B93160", COLOR_SECONDARY: "#D75281" },
+  { COLOR_ID: 8, COLOR_PRIMARY: "#A5C9CA", COLOR_SECONDARY: "#E7F6F2" },
+  { COLOR_ID: 9, COLOR_PRIMARY: "#6E85B7", COLOR_SECONDARY: "#B2C8DF" },
+  { COLOR_ID: 10, COLOR_PRIMARY: "#FBA1A1", COLOR_SECONDARY: "#FBC5C5" },
+];
```

Comment:
We are not using COLOR_PRIMARY or COLOR_SECONDARY, can we remove it?

Reply:
we would be using `/colors` api, in the future, wouldn't we require it then?